docs/testing.md

```diff
@@ -32,27 +32,63 @@ Those can be ignored and `/skip`ed. In example:
 ```
 Upstream issue: https://github.com/openshift/ci-operator-prowgen/issues/79 
 
+## Verify 
+
+This make target runs a series of checks to **verify** if it complies with standard Golang conventions and practices.  This can be run **locally** with the command `make verify`.  When creating a pull-request, this test is part of the acceptance criteria. 
+
 ## Unit test
 
 Packages under `pkg` directory contain their individual unit tests.
-To run all unit tests locally, execute: `make unit` or `go test ./...`
+
+To run all unit tests **locally**, execute: `make unit`
+
+_Note: This will **generate** bindata.go files when using this Makefile target as well as testinsights._
```

Comment:
Hm. No changes to `bindata.go` should appear if you do not change files we including in bindata.

For example, if you checkout the repo and run `make unit` or any other command that depends on `make generate` you should not see any new changes.

Reply:
You probably see it because you are still on pinned go-bindata version. Try update switch to the latest go-bindata.

Reply:
Sorry, I probably just misunderstood the sentence. Looks good to me.

/lgtm